packages/components/src/components/scrollable/scrollable.stories.ts

```diff
@@ -43,148 +47,147 @@ export default {
+    <sd-scrollable orientation="vertical" buttons scrollbars>
+      <div class="items-start h-max p-4 text-sm justify-start">
+        <p>Scroll vertically</p>
+        <br />
+        <p>This is a long scrollable content.</p>
+        <p>It contains multiple paragraphs and lines.</p>
+        <p>The content is intentionally long to trigger scrolling. You can scroll horizontally and vertically.</p>
+        <p>The scrollable component will display shadows and buttons based on the props.</p>
+        <p>Customize the content and attributes as needed.</p>
+      </div>
+    </sd-scrollable>
+  `
 };
 
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless">
-      ${generateTemplate({ args, constants: [{ type: 'attribute', name: 'buttons', value: true }] })}
-    </div>`;
-  },
+/**
+ * Use the `buttons` attribute to display buttons indicating scrollable content.
+ */
 
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-scrollable');
-    await waitUntil(() => el?.shadowRoot?.querySelector('button'));
+export const Buttons = {
+  name: 'Buttons',
+  render: () => html`
+    <sd-scrollable orientation="vertical" buttons>
+      <div class="items-start h-max p-4 text-sm justify-start">
+        <p>Scroll vertically</p>
+        <br />
+        <p>This is a long scrollable content.</p>
+        <p>It contains multiple paragraphs and lines.</p>
+        <p>The content is intentionally long to trigger scrolling. You can scroll horizontally and vertically.</p>
+        <p>The scrollable component will display shadows and buttons based on the props.</p>
+        <p>Customize the content and attributes as needed.</p>
+      </div>
+    </sd-scrollable>
+  `
+};
 
-    el?.shadowRoot?.querySelector<HTMLElement>('button')!.focus();
-  }
+/**
+ * Use the `shadows` attribute to display visual cues indicating scrollable content.
+ */
+
+export const Shadows = {
+  name: 'Shadows',
+  render: () => html`
+    <sd-scrollable orientation="vertical" shadows>
+      <div class="items-start h-max p-4 text-sm justify-start">
+        <p>Scroll vertically</p>
+        <br />
+        <p>This is a long scrollable content.</p>
+        <p>It contains multiple paragraphs and lines.</p>
+        <p>The content is intentionally long to trigger scrolling. You can scroll horizontally and vertically.</p>
+        <p>The scrollable component will display shadows and buttons based on the props.</p>
+        <p>Customize the content and attributes as needed.</p>
+      </div>
+    </sd-scrollable>
+  `
+};
+
+/**
+ * Use the `inset` attribute to adjust the inset padding.
+ */
+
+export const Inset = {
+  name: 'Inset',
+  render: () => html`
+    <sd-scrollable orientation="vertical" inset>
+      <div class="items-start h-max p-4 text-sm justify-start">
+        <p>Scroll vertically</p>
+        <br />
+        <p>This is a long scrollable content.</p>
```

Comment:
Use same text like in Figma

Union Investment has extended its lease with Converse, the footwear and apparel brand known for its deep connection to youth culture, at 1 Lovejoy Wharf in Boston. The contract for around 20,000 sqm of state-of-the-art office space was renewed long-term. The building has been part of the UniImmo: Global open-ended real estate fund portfolio since 2016 and is the global headquarters of the lifestyle brand, which has been part of NIKE, Inc. since 2003.